Code/Framework/AzCore/AzCore/Asset/AssetCommon.h

```diff
@@ -157,6 +157,7 @@ namespace AZ
             AZ_FORCE_INLINE const AssetId& GetId() const { return m_assetId; }
             AZ_FORCE_INLINE const AssetType& GetType() const { return RTTI_GetType(); }
             AZ_FORCE_INLINE int GetUseCount() const { return m_useCount.load(); }
+            int GetCreationToken() const { return m_creationToken; }
```

Comment:
Should it be consistent in the use of AZ_FORCE_INLINE as the rest? or should AZ_FORCE_INLINE be removed while you're here?

Reply:
I can remove the existing ones